src/v/http/client.cc

```diff
@@ -44,6 +44,11 @@ namespace http {
 
```

Comment:
> AWS S3 uses 10s timeout for all https connections. S3 connection
pool in redpanda didn't use any timeouts before this commit.
...

Amazing commit message!

Reply:
+1